curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efc4f528d6a74d05e68af74.md

```diff
@@ -9,7 +9,11 @@ dashedName: step-56
 
 There's another way to associate an `input` element's text with the element itself. You can nest the text within a `label` element and add a `for` attribute with the same value as the `input` element's `id` attribute.
 
-Associate the text `Loving` with the checkbox by only nesting the text `Loving` in a `label` element and place it to the right side of the checkbox `input` element.
+Associate the text Loving with the checkbox by nesting only the text Loving in a label element and giving it an appropriate for attribute
+
+```html
+  <label for="sweet">Sweet</label> <input id="sweet">
+```
```

Comment:
Thank you for following the guideline, before adding example, we want to test the new text.

```suggestion
Associate the text `Loving` with the checkbox by nesting only the text `Loving` in a `label` element and giving it an appropriate `for` attribute.
```

Reply:
Agreed, I think we might be able to avoid the confusion with the wording change alone!

Reply:
So how did it go for you guys ?

Reply:
I see how my wording cased confusion, sorry about that.

We want to add the words without an example to the repo as a test first.

 If it isn't enough we can open new issue for the example, in the future.

Reply:
Ah ok. 
Considered this case closed ?